cli/enry/main.go

```diff
@@ -136,6 +139,50 @@ func printPercents(out map[string][]string, buff *bytes.Buffer) {
 	}
 }
 
+func printFile(file string, buff *bytes.Buffer) {
+	content, err := ioutil.ReadFile(file)
+	if err != nil {
+		fmt.Println(err)
+	}
+	totalLines, sloc := getLines(file, content)
+	fileType := getFileType(file, content)
+	language := enry.GetLanguage(file, content)
+	mimeType := enry.GetMimeType(language)
+
+	buff.WriteString(fmt.Sprintf("%s: %d lines (%d sloc)\n", filepath.Base(file), totalLines, sloc))
+	buff.WriteString(fmt.Sprintf("  type:      %s\n", fileType))
+	buff.WriteString(fmt.Sprintf("  mime type: %s\n", mimeType))
+	buff.WriteString(fmt.Sprintf("  language:  %s\n", language))
+}
+
+func getLines(file string, content []byte) (int, int) {
```

Comment:
* sloc is a confunsing name, since it's ill-defined (not sure if `sloc` is consistently used in enry to refer to non-blank lines, in that case it'd be ok)
* Messing bytes and strings is confusing, I'd simplify it just using strings.

```go
func getNonBlankLines(file string, content string) (int, int) {
  totalLines := strings.Count(content, "\n")
  blankLines := strings.Count(content, "\n\n")
  return totalLines, totalLines - blankLines
}

totalLines, nonBlackLines := getLines(file, content.String())
```